pre_commit/languages/helpers.py

```diff
@@ -64,5 +70,21 @@ def target_concurrency(hook):
                 return 1
 
 
+def _shuffled(seq):
+    """Deterministically shuffle identically under both py2 + py3."""
+    fixed_random = random.Random()
+    if six.PY2:  # pragma: no cover (py2)
+        fixed_random.seed(FIXED_RANDOM_SEED)
+    else:
+        fixed_random.seed(FIXED_RANDOM_SEED, version=1)
+
+    seq = list(seq)
+    random.shuffle(seq, random=fixed_random.random)
```

Comment:
You have to do kind of a weird dance to get these to shuffle identically between py2 and py3 -- `fixed_random.shuffle` won't work because in py3 `random.shuffle` picks a different algorithm than what's used in py2 if you don't specify `random` as a kwarg; `random.sample` which would probably be preferred to avoid turning it into a list and mutating it has a similar problem.

We probably don't really need it to be identical between py2 and py3, but it means we don't need to go fix a bunch of tests. Idk, worth it?